src/bio/bio_xstream.c

```diff
@@ -159,6 +159,25 @@ bypass_health_collect()
 	return nvme_glb.bd_bypass_health_collect;
 }
 
+struct bio_faulty_criteria	glb_criteria;
+
+/* TODO: Make it configurable through control plane */
+static inline void
+set_faulty_criteria(void)
+{
+	glb_criteria.fc_enabled = false;
+	glb_criteria.fc_max_io_errs = 5;
```

Comment:
don't have much knowledge about SSD, but 5 looks too aggresive?

Reply:
I'm not sure too, let's adjust the number when we enable the auto faulty feature.